src/big/big_float.cr

```diff
@@ -293,3 +293,21 @@ module Math
     BigFloat.new { |mpf| LibGMP.mpf_sqrt(mpf, value) }
   end
 end
+
+# :nodoc:
+struct Crystal::Hasher
+  def float(value : BigFloat)
+    permute(float_normalize_wrap(value) do |value|
```

Comment:
Please assign the result of the `float_normalize_wrap` to a variable which is passed to `permute`. Having a multi-line block inside a method call arguments is just bad style.

Reply:
@akzhan You missed that one ;)

Reply:
@Sija Thanks!